plugin-server/src/cdp/templates/_destinations/tiktok_ads/tiktok.template.ts

```diff
@@ -197,21 +184,227 @@ if (res.status >= 400) {
             required: false,
         },
     ],
-    filters: {
-        events: [
-            { id: '$pageview', name: 'Pageview', type: 'events' },
-            { id: 'Payment Info Entered', type: 'events' },
-            { id: 'Product Added', type: 'events' },
-            { id: 'Product Added to Wishlist', type: 'events' },
-            { id: 'Product Clicked', type: 'events' },
-            { id: 'Order Completed', type: 'events' },
-            { id: 'Signed Up', type: 'events' },
-            { id: 'Checkout Started', type: 'events' },
-            { id: 'Order Completed', type: 'events' },
-            { id: 'Products Searched', type: 'events' },
-            { id: 'Product Viewed', type: 'events' },
-        ],
-        actions: [],
-        filter_test_accounts: true,
-    },
+    filters: { bytecode: ['_H', 1, 29] },
```

Comment:
If you are using mappings filters should be empty. In fact the bytecode should never be in the template